### DISCUSSION TITLE
Contributo al subentro in ANSC

### DISCUSSION THREAD

**Discussion by B86p** [italia/ansc#1139] [Q&A] [answered]
Se il contributo fornito dal ministero (che è a fondo perduto) è superiore a quanto richiede la software house per aggiornare il sistema informatico, è possibile che tale cifra venga destinata all'incremento del fondo decentrato? Grazie

**Comment by acolosimosogei** [comment, marked as answer]
Buongiorno, 

purtroppo non possiamo risponderle noi, abbiamo quindi girato la sua discussion a un ente adeguato.

Grazie e buona giornata,
Assistenza ANSC
